### PR TITLE
I need this pert file

### PR DESCRIPTION
**Pull request by jasonmotha**
Merging this guy from 6 months ago. But I think he needs a home.

### CODE REVIEW

**Review by jpa198 — DISMISSED** [review]
Looks good

File: srf_generation/velocity_model_generation/generate_3d_velocity_model_perturbation.py [hunk 1]
```diff
@@ -104,6 +104,8 @@ def generate_velocity_model_perturbation_file_from_config(
         for index, l_p in layer_params.items()
     ]
     try:
+        if n_processes == 1:
```
Comment (sungeunbae) [inline srf_generation/velocity_model_generation/generate_3d_velocity_model_perturbation.py:107]: It feels slightly odd to raise an assertion error for the default value for n_processes. I guess there is a good reason for this, and a comment will be helpful.

**Review by sungeunbae — CHANGES_REQUESTED** [review]
Good in general - a comment for n_process==1 assertion error will be all I'd like to have added

**Review by jpa198 — CHANGES_REQUESTED** [review]
All those changes to gen_3d seem like left over debugging changes? 
For a permanent change the stderr stuff should have a flag to enable stderr output and the n_processes thing should be changed to "if n_proc==1:  loop else: pool" and an error message saying to try again with n_proc=1 if it fails

**Comment by jasonmotha** [comment]
@sungeunbae @jpa198 There are a few more changes that need to be made to generate_3d_perturbations. So this PR is just going to be for the configuration file. Can you please review this once more?

**Review by sungeunbae — APPROVED** [review]

**Review by jpa198 — APPROVED** [review]

**Pull request merged**